frontend/src/state/ui.ts

```diff
@@ -103,6 +103,7 @@ export const DEFAULT_SEARCH_PARAMS = {
   page: 0,
   pageSize: 10,
   sorting: [] as SortingState,
+  quickSearch: '',
```

Comment:
I believe this is the only non-linting/formatting change applied to this PR @jvorcak - would you please elaborate on how the `quickSearch` param works to prevent changing URL in topic details page?

Reply:
@malinskibeniamin it's not the only change. Perhaps you missed the changes in `Tab.Messages/index`? Github hides them in a diff, because it's too big. 

In this PR, we change the way how we read parameters completely.